Automattic/wp-calypso
# Checkout: Add Plan upsell during checkout for free users

#### Changes proposed in this Pull Request

This PR adds the Personal plan upsell in checkout for free users which will be purchasing a domain during the test we'll run here: p99Zz8-10j-p2

Depends on D39094-code

#### Testing instructions

After applying D39094-code:

- Add a domain to cart
- Start checkout
- During the "Domain Contact Information" step, and in the last step of the checkout, you should see the following to the right of your screen:

<img width="280" alt="Screenshot 2020-02-21 at 1 02 42 PM" src="https://user-images.githubusercontent.com/13062352/75063063-5eeba680-54e4-11ea-96e4-c0af6df39908.png">

Please verify that the message shown in this box is always correct depending on the price of the domain (cheaper than the plan; same price as the plan; more expensive than the plan)
